### DISCUSSION TITLE
Meta_Hours_13

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#3697] [Meta-Hours]
# Light mode 💡
Meanwhile, we have the white version of KodaDot with a **few** changes, which would help attract a new audience who have been asking for light mode.

More release notes you can be read in recent release [Light mode at midnight](https://github.com/kodadot/nft-gallery/releases/tag/0.0.17)

Feel free to take a look and do click through; we will be making new tutorials and refreshing as we have there few major improvements and usual bugfixes in interactions.

The major highlight is the **super speedy search** 🚅🔎  powered by SubSquid! 
Go and try it yourself! -> [kodadot.xyz](https://kodadot.xyz)  

![image](https://user-images.githubusercontent.com/5887929/185143836-ea88e8e6-48d8-425e-938d-8d3c3e795aaa.png)

# Re-org
We had and will have some changes in KodaDot; a more verbose version is here 
- #3729 

# Refactoring components plus redesign
<img width="750" alt="image" src="https://user-images.githubusercontent.com/5887929/183412174-06d7045c-2642-4161-8741-03c5f7f2387d.png">

We will have a new UI coming along. We've decided to go with minimalistic neo-brutalism, which feels much more complete than current, emphasising end-user interaction. The final design at freeze is circulating among issues and frontend folks who will be putting their hands down on that to make it real. 
The first implemented version would be the landing page, then gallery item detail with collections and so on. I bet it will be done accomplished in 6-8 weeks with the majority of working components, maybe even sooner, who knows. 
I believe we will be productive in SubWork:)
More you can read here
- #3710 

# Kusama Treasury
Finally, it's time to make a treasury proposal for Kusama Treasury to fund our efforts and drive things faster. Cross fingers everything plays out in our favour :)
- https://github.com/kodadot/nft-gallery/issues/2072

# Basilisk - Rococo 
I bet @vikiival could speak about this as we will be coming close to launching the Rococo testing phase. Last phase before we go live on real Basilisk.
- #3376 

# MoonRiver & MoonBeam
Yes, KodaDot EVM is will real. We plan to support MoonRiver and Moonbeam after we've [hacked Moonsama into KodaDot](https://beta.kodadot.xyz/moonsama) successfully. 
If you have an EVM collection based on MoonRiver or Moonbeam, please feel free to fill up the form [form.kodadot.xyz/evm](https://form.kodadot.xyz/evm) so we can list you on KodaDot among the first ones! 
- https://github.com/kodadot/nft-gallery/issues/3647

# Roadmap
As this always changes, you can peak at our progress and issues 
[in the bigger picture at our Roadmap Q3Q4](https://github.com/orgs/kodadot/projects/4/views/1)

- currently [working on integrating RMRK 2.0 into KodaDot](https://github.com/kodadot/packages/pull/43)

# Time off
As we are working from a vacation destination [at Bled in Slovenia in our co-working](https://subwork.xyz), it's time for us to unplug. I bet before anticipated ETHBERLIN^3 (where we will be hacking again), we will gonna unplug and try to turn off our laptops and prepare for launching our partnerships :)

<img src="https://media3.giphy.com/media/3ohzdJqa9Lx5Gs5oY0/giphy.gif"/>

Probably pretty much that's it. 
Feel free to drop your updates 👀

**Comment by vikiival** [comment]
**Viki's Meta Minutes**
-

### DEV in numbers:
`is:pr is:merged created:>=2022-07-20`

> 123 PRs merged

### Internal:

With @yangwao we went through a long term vision of KodaDot and from technical perspective:

1. New desing will be written in Vue3

2. Codebase will become much smaller

3. App will get faster and better data analytics.

### FireSquid:

Since #3726 we are running a FireSquid on prod for RMRK NFTs on KodaDot.
The app is much smoother and also search is a bit more refined :)

In the similar manner we would migrate [kodadot/snek](https://github.com/kodadot/snek) once we release an implementation for Basilisk Rococo.

The migration for the EVM is in the process, there is just a small bug on the @subsquid side which should they fix soon ❤️.

### Basilisk:


New implementation of Basilisk is almost done ❤️ we need to check some possible edge cases which are not implemented.

What's new? 

- Multi Fee Payment
- Everything else in KSM

What will change? 

- We will sunset the current BSX Sandbox, 


### RMRK V2 support:

Since we are running FireSquid we are crafting a way how to make compatibility layer in rubick for RMRK v2 NFTs.



### EVM support:

Yes, as @yangwao mentioned we will have EVM support for Moonbeam network ❤️. We are curating a list of collections that will be running the first draft of the explorer.

Moreover we went through research to bring the modular marketplace to the EVM, so stay tuned ^-^--)


![RUNEVM_256](https://user-images.githubusercontent.com/22471030/185167762-8d09a10b-1b26-43c8-be6c-e6d4e13b5c98.png)


### Outside of the main repository:

KodaDot is powered by many [internal packages](https://github.com/kodadot/packages), [services](https://github.com/kodadot/workers) and of course indexers namely: [rubick](https://github.com/kodadot/rubick), [snek](https://github.com/kodadot/snek) and [click](https://github.com/kodadot/click). 

In these repositories you can see a ton of issues that would help us asymmetrically. 


### Highlights

I really want to thank @preschian that he automatised deploy of @subsquid indexer. Which saves my time (~ 10 min per deploy)

**Comment by DeluneClaire** [comment]
Claire's minute:
- Crowdcast session (ClownWorldHouse) - articles

- Yumi (a small sample)
![unknown](https://user-images.githubusercontent.com/104791453/185901252-ff5917bc-7963-4c7d-8a5a-ba9e99f363df.png)

- Ergione - T-shirt design
<img width="776" alt="Snímek obrazovky 2022-08-22 v 12 37 30" src="https://user-images.githubusercontent.com/104791453/185901972-94accb27-b05d-40a0-82dc-3a9e7f922098.png">

- Somonicav - ?? update
<img width="953" alt="Snímek obrazovky 2022-08-22 v 12 45 07" src="https://user-images.githubusercontent.com/104791453/185903358-9bd75ca2-85eb-4b60-8e21-b997761192a3.png">


- Kusama Dragons - New collection, Airdrops, Crowdcast sessions, Substack articles
![2022-08-22 12 38 11](https://user-images.githubusercontent.com/104791453/185902294-00e7deba-ba8a-49dd-bdd4-5a90dcd5c372.jpg)

- Yuliia.s collab: 1) Coins with stamp characters
                           2) Something like a comic book about her cat, Totoro (Manga storyboard)

**Comment by damskyftw** [comment]
Damsky’s minutes

- new juniors on the way
- wagmedia treasury 
- DOTBrno
